autovizwidget/autovizwidget/utils/configuration.py

```diff
@@ -0,0 +1,33 @@
+# Distributed under the terms of the Modified BSD License.
+from hdijupyterutils.constants import EVENTS_HANDLER_CLASS_NAME, LOGGING_CONFIG_CLASS_NAME
+from hdijupyterutils.utils import join_paths
+from hdijupyterutils.configuration import override as _override
+from hdijupyterutils.configuration import override_all as _override_all
+from hdijupyterutils.configuration import with_override
+
+from .constants import HOME_PATH, CONFIG_FILE
+    
+
+d = {}
+path = join_paths(HOME_PATH, CONFIG_FILE)
+
+
+def override(config, value):
+    global d, path
```

Comment:
Just tried this in a new Python script:

    d = {}
    path = 'hi'

    def override():
        print d, path

    override()

... and it works.